### PR TITLE
app-editors/leafpad: fdo-mime migration

### PR DESCRIPTION
**Pull request by hanetzer**
Also bumped EAPI to 6 and replaced eutils/epatch with a PATCHES=()
array, reformatted patches to apply.

Package-Manager: Portage-2.3.27, Repoman-2.3.9

### CODE REVIEW

File: app-editors/leafpad/leafpad-0.8.18.1.ebuild [hunk 1]
```diff
@@ -22,11 +23,10 @@ DEPEND="${RDEPEND}
 
 DOCS="AUTHORS ChangeLog NEWS README"
 
-src_prepare() {
-	epatch \
-		"${FILESDIR}"/${P}-fdo.patch \
-		"${FILESDIR}"/${P}-format-security.patch
-}
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.8.18.1-fdo.patch
```
Comment (a17r) [inline app-editors/leafpad/leafpad-0.8.18.1.ebuild]: `P` to `PN` is an unnecessary change.

Comment (hanetzer) [reply app-editors/leafpad/leafpad-0.8.18.1.ebuild]: Unneeded atm, sure, but  in the event this does get updated and the patch no longer applies it strikes me as simpler
to have `${PN}-version` already.
